### PR TITLE
bug/6800-Dylan-VetSatusLandscapeBackground

### PR DESCRIPTION
**Pull request by Sparowhawk**
## Description of Change
Removed insets on scrollview like onboarding to extend the background color to the edges on Landscape

## Screenshots/Video

Screenshot 2023-10-18 at 11 36 57 AM
Screenshot 2023-10-18 at 11 37 06 AM

## Testing
yarn test

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations
Another issue with the iOS white background thing after rotating from portrait to landscape. This was fixed for onboarding and the home screen ([4977](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/4977)).

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [x] PR is connected to issue(s)
- [x] Tests are included to cover this change (when possible)
- [x] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [x] No secrets or API keys are checked in
- [x] All imports are absolute (no relative imports)
- [x] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx [hunk 1]
```diff
@@ -83,69 +83,67 @@ const VeteranStatusScreen: FC<VeteranStatusScreenProps> = () => {
...
-        <Box mx={theme.dimensions.gutter}>
-          {ratingIsDefined && (
-            <Box {...boxProps}>
-              <TextView variant="MobileBodyBold" color="primaryContrast">
-                {t('disabilityRating.title')}
-              </TextView>
-              <TextView variant="MobileBody" color="primaryContrast" testID="veteranStatusDisabilityRatingTestID">
-                {combinedPercentText}
+    <LargePanel title={t('veteranStatus.title')} rightButtonText={t('close')} dividerMarginBypass={true} removeInsets={true} testID="veteranStatusTestID">
+      <Box mx={isPortrait ? theme.dimensions.gutter : theme.dimensions.headerHeight} alignItems="center" mt={theme.dimensions.standardMarginBetween}>
```
Comment (dumathane) [inline VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx:87]: Why are we changing the X margin from 20 to 64 when in landscape? Was that defined somewhere?

File: VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx [hunk 2]
```diff
@@ -83,69 +83,67 @@ const VeteranStatusScreen: FC<VeteranStatusScreenProps> = () => {
...
+        <VAIcon name={'Logo'} />
```
Comment (dumathane) [inline VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx:88]: dropped the testid?

File: VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx [hunk 3]
```diff
@@ -83,69 +83,67 @@ const VeteranStatusScreen: FC<VeteranStatusScreenProps> = () => {
...
+        {/* <Box my={theme.dimensions.standardMarginBetween}>
+        //TODO: Put back PhotoUpload later after concerns have been met
+          <PhotoUpload width={100} height={100} />
+        </Box> */}
+        <Box my={theme.dimensions.formMarginBetween}>
+          <TextView textTransform="capitalize" mb={theme.dimensions.textIconMargin} variant="BitterBoldHeading" color="primaryContrast">
```
Comment (dumathane) [inline VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx:94]: removed testID?

File: VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx [hunk 4]
```diff
@@ -83,69 +83,67 @@ const VeteranStatusScreen: FC<VeteranStatusScreenProps> = () => {
...
+            {personalInfo?.fullName}
+          </TextView>
+          {accessToMilitaryInfo && (
+            <Box display="flex" flexDirection="row">
+              {getBranchSeal()}
+              <TextView ml={10} variant="MobileBody" color="primaryContrast">
```
Comment (dumathane) [inline VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx:100]: removed testid?

File: VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx [hunk 5]
```diff
@@ -83,69 +83,67 @@ const VeteranStatusScreen: FC<VeteranStatusScreenProps> = () => {
...
-            {getPeriodOfService}
-          </Box>
-          <Box {...boxProps} borderBottomWidth={theme.dimensions.borderWidth} mb={theme.dimensions.formMarginBetween}>
-            <TextView variant="MobileBodyBold" color="primaryContrast">
-              {t('personalInformation.dateOfBirth')}
+              {t('disabilityRating.title')}
             </TextView>
-            <TextView variant="MobileBody" color="primaryContrast" testID="veteranStatusDOBTestID">
-              {personalInfo?.birthDate || t('personalInformation.informationNotAvailable')}
+            <TextView variant="MobileBody" color="primaryContrast">
```
Comment (dumathane) [inline VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx:113]: removed testid?

File: VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx (hunk 2 above)
Comment (Sparowhawk) [reply VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx:88]: good catch, the merge conflict with Rachel's was a tricky one.

File: VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx (hunk 3 above)
Comment (Sparowhawk) [reply VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx:94]: good catch, the merge conflict with Rachel's was a tricky one.

File: VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx (hunk 4 above)
Comment (Sparowhawk) [reply VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx:100]: good catch, the merge conflict with Rachel's was a tricky one.

File: VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx (hunk 5 above)
Comment (Sparowhawk) [reply VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx:113]: good catch, the merge conflict with Rachel's was a tricky one.

**Review by dumathane — APPROVED** [review]

**Review by theodur — APPROVED** [review]

**Review by rbontrager — APPROVED** [review]

**Pull request merged**